ga4gh/datamodel/variants.py

```diff
@@ -407,7 +407,8 @@ def generateVariantAnnotation(self, variant):
         ann.variantId = variant.id
         ann.start = variant.start
         ann.end = variant.end
-        ann.created = int(datetime.datetime.now().strftime("%s"))
+        ann.created = datetime.datetime.now().strftime(
+            "%Y-%m-%dT%H:%M:%S.%fZ")
```

Comment:
What about using isoformat method? This definitely seems like something the standard library should do.